src/util/v16/v16_avx512.h

```diff
@@ -523,39 +523,41 @@ namespace v16
 
   // v16 memory manipulation functions
 
+  // Portable version.
```

Comment:
What's with these repeated comments saying `portable version`? (another one below)

Reply:
Sometimes, for the purposes of debugging or performance benchmarking, I will replace an intrinsics implementation with the portable implementation.  For the case of performance benchmarking, I want to convince myself that the intrinsics version produces a measurable increase in performance over the easy to read and understand portable version.  This is also an indication in my mind that the intrinsics implementation is not as complete and polished as I would like it to be.  At some point, I would like to feel that the intrinsics versions are as complete and as performant as they can be.  I should probably make another round of cleaning up the intrinsics headers and removing stuff like that, since of course, it already exists in the portable header files.  Right now, I don't have the time for more cleaning work but try to keep doing that on a steady basis.